types/chance/index.d.ts

```diff
@@ -51,7 +51,7 @@ declare namespace Chance {
         birthday(): Date;
         birthday(opts?: Options): Date | string;
         cf(opts?: Options): string;
-        cpf(): string;
+        cpf(opts?: { formatted?: boolean }): string;
```

Comment:
this would allow `chance.cpf({})` correct? although that won't technically break anything, i would take the `?` off of `formatted` in order to catch a scenario where someone isn't using the method as intended. the `AtLeastOneKey` catches that for more complex objects, but since this one is so simple, you can j/ drop the `?`.

```suggestion
        cpf(opts?: { formatted: boolean }): string;
```